src/main/groovy/com/sequenceiq/cloudbreak/client/CloudbreakClient.groovy

```diff
@@ -86,6 +86,14 @@ class CloudbreakClient {
                 group.append("},");
             }
         }
+        String params = "";
+        StringBuilder paramsBuilder = new StringBuilder();
+        for (Map.Entry<String, String> map : parameters.entrySet()) {
+            paramsBuilder.append(String.format("\"%s\" : \"%s\",", map.getKey(), map.getValue()));
+        }
+        if (paramsBuilder.toString().length() > 0) {
+            params = paramsBuilder.toString().substring(0, paramsBuilder.toString().length() - 1);
```

Comment:
paramsBuilder.toString().length() == 0 ? paramsBuilder.toString() : paramsBuilder.toString().substring(0, paramsBuilder.toString().length() - 1) or something like that is shorter.
Or please using something much 'groovier' code.